jetty-client/src/main/java/org/eclipse/jetty/client/AbstractConnectionPool.java

```diff
@@ -90,6 +109,17 @@ protected void doStop() throws Exception
         return CompletableFuture.allOf(futures);
     }
 
+    @ManagedAttribute(value = "The maximum duration in milliseconds a connection can be used for before it gets closed")
+    public long getMaxDuration()
+    {
+        return maxDuration;
+    }
+
+    public void setMaxDuration(long maxDuration)
+    {
+        this.maxDuration = maxDuration;
```

Comment:
How does this get set?  I don't see a way of configuring this (or a testcase demonstrating how to set it)

Reply:
I've now added a new parameter to `ConnectionPoolTest` that is running the test suite against a pool with a configured max duration.